taskcluster/utils.py

```diff
@@ -11,6 +11,8 @@
 
 MAX_RETRIES = 5
 
+CONFIG_FILE = os.path.expanduser('~/.config/taskcluster.json')
```

Comment:
Maybe just use: https://pypi.python.org/pypi/appdirs
It's a small dependency, from what I can see it's pure python... There's probably libs with better XDG compliance... but I doubt they are cross platform, I saw some solutions dependent glib...

Note: `os.path.expanduser('~/.config/taskcluster.json')` works great on linux and mac, but it is less nice on windows.